clients/fides-js/src/components/ConsentButtons.tsx

```diff
@@ -45,6 +45,7 @@ export const ConsentButtons = ({
   isGVLLoading,
 }: ConsentButtonProps) => {
   const [isLoadingModal, setIsLoadingModal] = useState<boolean>(false);
+  const [isLanguageMenuOpen, setIsLanguageMenuOpen] = useState(false);
```

Comment:
I'm not too familiar with the nuances of TS, is there a significant difference between including <boolean> to indicate the type or not?

Reply:
nah, I should add, though, to be explicit about types 👍